frontend/ui/widget/container/inputcontainer.lua

```diff
@@ -304,29 +305,50 @@ end
 --       [1] The most common implementation you'll see is a NOP for ReaderUI modules that defer gesture handling to ReaderUI.
 --           Notification also implements a simple one to dismiss notifications on any user input,
 --           which is something that doesn't impede our goal, which is why we don't need to deal with it.
-function InputContainer:onIgnoreTouchInput(toggle)
-    local Notification = require("ui/widget/notification")
-    if toggle == false then
+function InputContainer:setIgnoreTouchInput(state)
+    logger.dbg("InputContainer:setIgnoreTouchInput", state)
+    if state == true then
+        -- Replace the onGesture handler w/ the minimal one if that's not already the case
+        if not InputContainer._onGesture then
+            InputContainer._onGesture = InputContainer.onGesture
+            InputContainer.onGesture = InputContainer._onGestureFiltered
+            -- Notify UIManager so it knows what to do if a random popup shows up
+            UIManager._input_gestures_disabled = true
+            logger.dbg("Disabled InputContainer gesture handler")
+
+            -- Notify our caller that the state changed
+            return true
+        end
+    elseif state == false then
         -- Restore the proper onGesture handler if we disabled it
         if InputContainer._onGesture then
             InputContainer.onGesture = InputContainer._onGesture
             InputContainer._onGesture = nil
-            Notification:notify("Restored touch input")
+            UIManager._input_gestures_disabled = false
+            logger.dbg("Restored InputContainer gesture handler")
+
+            return true
         end
-    elseif toggle == true then
-        -- Replace the onGesture handler w/ the minimal one if that's not already the case
-        if not InputContainer._onGesture then
-            InputContainer._onGesture = InputContainer.onGesture
-            InputContainer.onGesture = InputContainer._onGestureFiltered
+    end
+
+    -- We did not actually change the state
+    return false
+end
+
+-- And the matching Event handler
+function InputContainer:onIgnoreTouchInput(toggle)
+    local Notification = require("ui/widget/notification")
+    if toggle == true then
+        if self:setIgnoreTouchInput(true) then
             Notification:notify("Disabled touch input")
         end
+    elseif toggle == false then
+        if self:setIgnoreTouchInput(false) then
+            Notification:notify("Restored touch input")
```

Comment:
*ahem* ;-)
```suggestion
            Notification:notify(_("Restored touch input"))
```

Reply:
Oh, this isn't new; I thought I'd managed to magically remove gettext calls somehow :D.